src/test/kotlin/racingcar/domain/CarKoTest.kt

```diff
@@ -0,0 +1,30 @@
+package racingcar.domain
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.ints.shouldBeExactly
+import io.mockk.every
+import io.mockk.mockk
+import racingcar.RandomNumberUtils
+
+class CarKoTest : DescribeSpec({
+    describe("자동차 이동 테스트") {
+        val randomNumberGenerator = mockk<RandomNumberUtils>()
+
+        val car = Car("테스트 자동차")
+        beforeEach { car.position = 0 }
```

Comment:
하나의 car 객체를 매 테스트 케이스마다 공유해서 사용하는 것 보다는 각 테스트 케이스마다 car 객체를 재생성하도록 변경해보면 좋을 것 같아요. 😃 

```suggestion
        lateinit var car: Car

        beforeEach { car = Car("테스트 자동차") }
```

Reply:
피드백을 반영하다보니 자연스레 없어진 코드였지만, 의도에 대해서 말씀드려 봅니다!

beforeEach를 사용했던 이유는, 테스트 시 여러 객체를 initiating하는 것이 비효율 적이라고 생각하여서 입니다. 
(사실 크리티컬하게 퍼포먼스에 영향을 주지는 않을 것 같지만요..ㅎㅎ)
경록님께서 위와 같은 리뷰를 주셨던 이유는 어떤 측면에서였을까요 !? 궁금해서 여쭤봅니다 !!! 😄